habitat_baselines/config/objectnav/ddppo_objectnav.yaml

```diff
@@ -0,0 +1,60 @@
+BASE_TASK_CONFIG_PATH: "configs/tasks/objectnav_mp3d.yaml"
+TRAINER_NAME: "ppo"
+ENV_NAME: "NavRLEnv"
+SIMULATOR_GPU_ID: 0
+TORCH_GPU_ID: 1
+VIDEO_OPTION: ["disk", "tensorboard"]
+TENSORBOARD_DIR: "/checkpoint/maksymets/logs/habitat_baselines/ddppo/obj_nav_mp3d"
+VIDEO_DIR: "data/video_dir"
+TEST_EPISODE_COUNT: 2
```

Comment:
Let's put the actual number of val episodes here.  It is hard to figure out what number should go here when you want to evaluate on everything otherwise.